deployments/datahub/images/default/connectors/2021-fall-phys-188-288.bash

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+
+cd ${CONDA_DIR}
+git clone https://github.com/HerculesJack/assignment-1910
+cd assignment-1910
+bash install.sh
```

Comment:
```suggestion
bash install.sh > /dev/null
```

Reply:
@balajialg @felder I think this might make the output log a bit better?

I think this is ok to merge, but let's set a 'back out date' when we can remove this. This should be published as a package on pypi.org and installed for the next time - I think we should try to stay away from just running arbitrary scripts.